charaData.json

```diff
@@ -659,6 +659,50 @@
         "imageURL": "./charaimgs/3040169000_01.png",
         "en": "Nier"
     },
+    "モニカ(リミテッドver)": {
+        "name": "モニカ(リミテッドver)",
+        "ja": "モニカ(リミテッドver)",
+        "element": "wind",
+        "type": "balance",
+        "race": "human",
+        "sex": "female",
+        "fav1": "sword",
+        "fav2": "none",
```

Comment:
Monika's fav2 is katana.
https://i0.wp.com/tkgranblue.com/wp-content/uploads/2019/08/2019-09-30-1.jpg